LANGUAGES/languages.json

```diff
@@ -360,51 +360,51 @@
         "name":"Turkish (Türkçe)",
         "FontFamily":"Courier New",
         "FontSize":16,
-        "Author":"Erhan Kültür/crasadure",
+        "Author":"Erhan Kültür/crasadure, Oğuz Ersen",
         
         "STR_ERROR":"Hata",
         "STR_WARNING":"Uyarı",
         "STR_INFO":"Bilgi",
         "STR_INCORRECT_DIR":"Lütfen doğru dizin altında çalıştırın!",
-        "STR_INCORRECT_TREE_DIR":"Dosyayı burada çalıştırma, lütfen yayınlanan en son kurulum paketini indirin ve oradan çalıştırın.",
+        "STR_INCORRECT_TREE_DIR":"Dosyayı burada çalıştırmayın, lütfen yayınlanan en son kurulum paketini indirin ve oradan çalıştırın.",
         "STR_DEVICE":"Aygıt",
         "STR_LOCAL_VER":"Ventoy GÜNCEL Sürüm",
-        "STR_DISK_VER":"Cihazdaki Ventoy Sürümü",
+        "STR_DISK_VER":"Aygıttaki Ventoy Sürümü",
```

Comment:
Tutarlı olması için tüm çeviri boyunca "aygıt" kelimesini kullandım. TDK'nın sözlüğüne göre "cihaz" Arapça kökenli olduğu için "aygıt" kelimesini kullanıyorum ama "cihaz" kullanacak şekilde de değiştirebilirim.